Packs/Slackbot_V2/Integrations/SlackBot/SlackBot.py

```diff
@@ -0,0 +1,920 @@
+"""
+
+
+def get_xsoar_list(list_name):
+    try:
+        api_response = demisto_client.generic_request_func(self=api_instance, path="/xsoar/lists", method="GET")
+        return (find_list_by_listname(api_response[0], list_name))
+    except ApiException as e:
+        logging.error("Exception when calling generic api: %s\n" % e)
+
+
+"""
+Helper function to get channel id for uploading of files.
+"""
+
+
+def get_channel_id(channel_name, client):
+    try:
+        # Call the conversations.list method using the WebClient
+        response = client.conversations_list()
+        demisto.debug(f"Result from conv:{response}")
+
+        # Check the channels in the response
+        for channel in response["channels"]:
+            if channel["name"] == channel_name:
+                return channel["id"]
+
+        logging.info("Channel not found")
+        demisto.debug("Channel not found")
+        return None
+
+    except Exception as e:
+        demisto.debug(f"Error fetching channels: {e}")
+        logging.error(f"Error fetching channels: {e}")
+        return None
+
+
+"""
+    Helper function used to load the success message XSOAR JSON list template and replace dynamic values to send back a custom Slack message to user/channel.
+
+    Args:
+      created_incident: list -> List object returned from running the demisto.createIncidents() command.
+"""
+
+
+def craft_incident_trigger_success_message(created_incident):
+    try:
+        # Define incident variables from created incident
+        incident_name = created_incident[0].get("name")
+        incident_id = created_incident[0].get("id")
+        if not created_incident[0].get("type"):
+            incident_type = "Unclassified"
+        else:
+            incident_type = created_incident[0].get("type")
+        submitted_by = json.loads(created_incident[0].get("rawJSON")).get("user").get("username")
+        created_at = created_incident[0].get("created")
+        incident_status = map_status_code(created_incident[0].get("status"))
+
+        # Load the success message XSOAR list template
+        success_message = get_xsoar_list(list_name="SlackIncidentTriggerSuccess")
+
+        # Parse through template and replace dynamic variables
+        success_message = json.loads(success_message)
+        success_message[2]["fields"][0]["text"] = f"*Incident Name:*\n{incident_name}"
+        success_message[2]["fields"][1]["text"] = f"*Incident ID:*\n{incident_id}"
+        success_message[3]["fields"][0]["text"] = f"*Incident Type:*\n{incident_type}"
+        success_message[3]["fields"][1]["text"] = f"*Submitted By:*\n@{submitted_by}"
+        success_message[4]["fields"][0]["text"] = f"*Created At:*\n{created_at}"
+        success_message[4]["fields"][1]["text"] = f"*Incident Status:*\n{incident_status}"
+
+        return success_message
+
+    except Exception as e:
```

Comment:
This might be too broad of an exception here. It would be better to wrap the problematic areas and catch those errors where they are so it's easier to debug.